android/app/src/androidTest/java/com/microsoft/codepush/common/utils/UtilsAndroidTest.java

```diff
@@ -0,0 +1,183 @@
+package com.microsoft.codepush.common.utils;
+
+import com.microsoft.codepush.common.exceptions.CodePushMalformedDataException;
+import com.microsoft.codepush.common.testutils.CommonFileTestUtils;
+
+import junit.framework.Assert;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.InputStream;
+
+import static com.microsoft.codepush.common.testutils.CommonFileTestUtils.getRealNamedFileWithContent;
+import static com.microsoft.codepush.common.testutils.CommonFileTestUtils.getTestingDirectory;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * This class tests all the {@link CodePushUtils} scenarios.
+ */
+public class UtilsAndroidTest {
+
+    /**
+     * Sample class for JSON mapping.
+     */
+    private final class SampleObject {
+        public String id;
+        public String name;
+
+        public SampleObject(String id) {
+            this.id = id;
+        }
+
+        public SampleObject(String id, String name) {
+            this.id = id;
+            this.name = name;
+        }
+    }
+
+    /**
+     * {@link CodePushUtils} instance.
+     */
+    private CodePushUtils mUtils;
+
+    @Before
+    public void setUp() {
+        FileUtils fileUtils = FileUtils.getInstance();
+        mUtils = CodePushUtils.getInstance(fileUtils);
+    }
+
+    /**
+     * Tests getting json object from correct json file.
+     */
+    @Test
+    public void testGetJsonObjectFromCorrectJsonFile() throws Exception {
+        String inputJsonString = "{\"key\":\"value\"}";
```

Comment:
I think this and below should be final constants, as done in other files.

Reply:
@annakocheshkova My opinion here is that test should be as obvious as possible and this class doesn't contain complex test case/mocks so no need to overload test class.